chapters/2-collaboration.tex

```diff
@@ -417,7 +417,7 @@ \subsection{Simplifying work across file formats}
 This decision will also affect the version control systems that your team can use.
 
 
-\subsection{Setting up a version control system}
+\subsection{Setting up version control}
```

Comment:
```suggestion
\subsection{Using version control}
```
Minor, but I think "using" is more appropriate than "setting up" as you use systems that are already set up